typed/symdiff_test.go

```diff
@@ -469,36 +466,19 @@ var symdiffCases = []symdiffTestCase{{
 }}
 
 func (tt symdiffTestCase) test(t *testing.T) {
-	var s schema.Schema
-	err := yaml.Unmarshal([]byte(tt.schema), &s)
-	if err != nil {
-		t.Fatalf("unable to unmarshal schema")
-	}
-
+	parser := framework.NewParserOrDie(tt.schema)
 	for i, quint := range tt.quints {
 		quint := quint
 		t.Run(fmt.Sprintf("%v-valid-%v", tt.name, i), func(t *testing.T) {
 			t.Parallel()
 
-			lhs, err := value.FromYAML([]byte(quint.lhs))
-			if err != nil {
-				t.Fatalf("unable to interpret lhs yaml: %v\n%v", err, quint.lhs)
-			}
-			t.Logf("parsed lhs object:\v%v", lhs.HumanReadable())
-
-			rhs, err := value.FromYAML([]byte(quint.rhs))
-			if err != nil {
-				t.Fatalf("unable to interpret rhs yaml: %v\n%v", err, quint.rhs)
-			}
-			t.Logf("parsed rhs object:\v%v", rhs.HumanReadable())
-
-			tvLHS := AsTypedUnvalidated(lhs, &s, tt.rootTypeName)
-			tvRHS := AsTypedUnvalidated(rhs, &s, tt.rootTypeName)
+			tvLHS := parser.FromYAMLOrDie(quint.lhs, tt.rootTypeName)
```

Comment:
OrDie will make the test super annoying if it fails, no?

Reply:
Yeah, it's not the end of the world, but it also doesn't hurt to check the error ...